externs/shaka/manifest.js

```diff
@@ -136,7 +136,8 @@ shakaExtern.InitDataOverride;
  *   audioRobustness: string,
  *   videoRobustness: string,
  *   serverCertificate: Uint8Array,
- *   initData: Array.<!shakaExtern.InitDataOverride>
+ *   initData: Array.<!shakaExtern.InitDataOverride>,
+ *   defaultKID: ?string
```

Comment:
There can be (and best practice for content authoring dictates there should be) multiple key IDs in a single manifest.  If you're going to go this route, this should be an array of strings (default empty list, not null), and it should be called keyIds.